CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## v0.13.0
+
+The primary motivation for this release is to add support for fetching images on the Docker Registry 2.0. It also includes other small improvments.
```

Comment:
Although since this is only enabled if you have the headers at build time, maybe we shouldn't mention it for now and wait until https://github.com/coreos/rkt/issues/1815 is closed.